app/controllers/orders/view.js

```diff
@@ -1,17 +1,28 @@
 import Controller from '@ember/controller';
-import { inject as service } from '@ember/service';
+
 
 export default Controller.extend({
   isLoading : false,
-  printThis : service(),
   actions   : {
     downloadInvoice() {
-      const selector = '.print';
-      const options = {
-        header     : 'Order Invoice',
-        printDelay : 800
-      };
-      this.get('printThis').print(selector, options);
+      this.set('isLoading', true);
+      this.get('loader')
+        .downloadFile(`/orders/invoices/${this.get('model.order.identifier')}`)
+        .then(res => {
+          const anchor = document.createElement('a');
+          anchor.style.display = 'none';
+          anchor.href = URL.createObjectURL(new Blob([res], { type: 'application/pdf' }));
+          anchor.download = 'Invoice.pdf';
+          anchor.click();
+          this.get('notify').success(this.get('l10n').t('Here is your Order Invoice'));
+        })
+        .catch(e => {
+          console.warn(e);
```

Comment:
don't disable the print thing for now, put it inside the catch block, if for some reason the above fails, it will be a fallback invoice.